app/controllers/users_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     new_user = User.new(user_params)
 
     if new_user.save
-      flash[:success] = "Hello #{new_user.first_name} #{new_user.last_name}, thank you for creating an account!"
+      flash[:success] = "Hello #{new_user.first_name}, thank you for creating an account!"
```

Comment:
Spending so much time working on and thinking about UX got me thinking about some of the language like this and I just tweaked it to make it seem more friendly. Changing "register" to "sign up" is another, probably better, example.